src/Tmds.DBus/DBusConnection.cs

```diff
@@ -711,9 +711,13 @@ private async void HandleMethodCall(Message methodCall, IMessageStream peer)
             if (_methodHandlers.TryGetValue(methodCall.Header.Path.Value, out methodHandler))
             {
                 var reply = await methodHandler(methodCall);
-                reply.Header.ReplySerial = methodCall.Header.Serial;
-                reply.Header.Destination = methodCall.Header.Sender;
-                SendMessage(reply, peer);
+                // Only reply if the caller expected it. Otherwise DBus will reject with org.freedesktop.DBus.Error.AccessDenied
```

Comment:
You can leave out the comment. The code is clear on its own.

Reply:
Comment removed